src/libsyntax/parse/lexer/tokentrees.rs

```diff
@@ -1,8 +1,9 @@
 use rustc_data_structures::fx::FxHashMap;
 use syntax_pos::Span;
 
+use super::{StringReader, UnmatchedBrace};
+
 use crate::print::pprust::token_to_string;
-use crate::parse::lexer::{StringReader, UnmatchedBrace};
```

Comment:
It's also not clear to me why these imports were changed?

Reply:
In https://github.com/rust-lang/rust/pull/65324 the module will eventually be moved to `syntax_parse::lexer` so making it relative is helpful towards that end.